README.md

```diff
@@ -55,7 +55,7 @@
    snippet-example-1.6.x.cs
    snippet-example-1.4.x.php
    snippet-example-1.7.x.php
-   snippet-example-1.8.x.py
```

Comment:
Hi @tiwarishubham635 , based on https://github.com/TwilioDevEd/api-snippets/pull/1047, I was hoping to see something about the Python upgrade in https://github.com/twilio/twilio-python/blob/main/UPGRADE.md. Is that correct?

Reply:
Yes, you are right. I have added the change in RC branch [here](https://github.com/twilio/twilio-python/pull/766/files#diff-434625111321abb225365806085a3753cb854d098ed9b4b6dc2001235c7f7bc4). Will be merging it soon. I will first get the twilio-python PR merged, and then after the successful release of python major version, this PR should be merged